aerogear/mobile-security-service
# fix(modal): reset textbox value when disable app modal closes (AEROGEAR-9056)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-9056

## What


Ensure that the textbox in the `DisableAppModal` always gets reset when the modal closes.

## Why


Expected behaviour.

## How

I moved the value from local state to Redux.

## Verification Steps
 
1. Click `Disable App`.
2. Enter a value into the textbox.
3. Close the modal, or click "Save" in the modal.
4. Click `Disable App` again.
5. The textbox should be empty again.

## Checklist:


## Progress
